graphql/src/execution/resolver.rs

```diff
@@ -72,69 +72,65 @@ pub trait Resolver: Clone + Send + Sync {
     /// Resolves entities referenced by a parent object.
     fn resolve_objects(
         &self,
-        parent: &Option<q::Value>,
+        objects_value: Option<q::Value>,
```

Comment:
I liked that this used to be called `parent` - it helped me in initially understanding this code to know that this object lives one level above where we currently are. It would be good to have a comment here what exactly we expect in the `q::Value`. The dynamic typing via `q::Value` makes it easy to lose track of what you are looking at.